Templates/index.md

```diff
@@ -0,0 +1,32 @@
+# How to Use EOS Doc Templates
+
+To submit your documentation to the EOS Network Foundation (ENF), create a fork of the receiving repository and then add work on your own fork. From there, do a PR from your fork to the main branch. DevRel then reviews the PR, suggests changes, if any, and after approval the PR can be merged. 
+
+ENF Templates use MarkDown language.
+
+Templates provide a structure for your content.
+
+Text identified within brackets [...] are instructions to you for the type of information to include in the topic.
+
+If a section title contains "(Optional)" and you don’t need to add information for that topic, add the description “N/A”.
+
+Please include your email or telegram address so DevRel can reach out to you with any questions.
+
+## How to Apply a Template to Your Content
+Complete the following steps to apply a template to your content:
+1. Open a new document in your .md editor
+
+2. Open the DevRel repository [Templates](../Templates/index.md) folder, or click a template link below, to open a template for your document type
+
+      a. [Overview](../Templates/Feature-Overview-Template.md): This template provides a quick outline of a feature. Links to more detailed information are provided at the bottom of the document.
+
+      b. [Explainer](../Templates/Feature-Explainer-Template.md):
+       This template is a guideline to use when documenting a feature. It provides information about the feature, how it fits into the big picture of the EOS blockchain as well as any dependencies and constraints. It also provides information on related use cases and expected results, as well as example code.
+
+      c. [Tutorial](../Templates/Tutorial-Template.md): This template provides an educational lesson with context. It contains one or more procedures to demonstrate mastery of a concept. A tutorial instructs users how to perform/achieve a particular goal. You need to add test data or tools necessary to complete and test the tutorial. A tutorial should contain 7 or less steps, if possible. If you have more than 7 steps, consider dividing the tutorial into two separate tutorials. Keep the tutorial experience for the user to an hour or less, if possible.
+      
+      d. [How To](../Templates/How-to-Template.md): This template provides step-by-step instructions to produce a desired result. It provides prerequisite information, when to use it,  as well as expected results. This template also provides an option for example code.
+4. Copy the contents of the template file
+5. Paste the contents of the template file into your new document and then write your content
```

Comment:
Btw, since these steps contain full sentences, shouldn't they end in a period (.) ?

Reply:
Done